gateway/src/inference/providers/tgi.rs

```diff
@@ -317,6 +311,54 @@ impl<'a> TGIRequest<'a> {
     }
 }
 
+#[derive(Clone, Debug, Default, PartialEq, Serialize)]
+#[serde(rename_all = "snake_case")]
+#[serde(tag = "type")]
+enum OpenAIResponseFormat {
```

Comment:
`TGIResponseFormat` and similar

Reply:
yep it looks good! now this is on me to get the TGI pod set up so we can try against a real instance and get this merged